.github/workflows/build.yaml

```diff
@@ -21,17 +21,17 @@ jobs:
         distribution: "adopt"
         java-package: "jdk"
         java-version: "${{ matrix.java }}"
-    - uses: "bufbuild/buf-setup-action@v1"
+    - uses: "bufbuild/buf-setup-action@v1.9.0"
```

Comment:
Don't we want to keep this at v1?

Reply:
for whatever reason that's what the README at buf-setup-action showed. It did seem weird they deviated from the established pattern

Reply:
@jzelinskie addressed in https://github.com/authzed/authzed-java/pull/32/commits/0c05f4f9b67f881ae5c82f8b0f8d7ea106a3c812